c/http.c

```diff
@@ -200,11 +203,32 @@ int writeFully(Socket *socket, char *buffer, int len){
       if (WRITE_FORCE && returnCode == EAGAIN) {
 #else
       if (WRITE_FORCE && returnCode == 1102) {
```

Comment:
We should be conditionally define'ing EAGAIN to 1102 on platforms where that's reasonable instead of hardcoding a magic value here.